modules/ROOT/pages/creating-a-skin.adoc

```diff
@@ -1,85 +1,108 @@
-:navtitle: Create a skin
-:description_short: Introducing skin creation.
-:description: Introducing skin creation, less and icon modification.
-:keywords: create, creator, skin, icon
+= Create a Skin for {productname}
+:navtitle: Create a Skin
+:description_short: Introduction to skin creation.
+:description: Learn how to create a custom skin, modify Less variables, and update icons for {productname}.
+:keywords: create, skin, customization, icon, Less
 
-This section provides information on how to manually create a new skin to customize the appearance of {productname} {productmajorversion}.
-
-The {productname} Skin Tool provides an interactive method for designing and creating a skin for {productname}, visit: http://skin.tiny.cloud/[http://skin.tiny.cloud/].
+This guide explains how to manually create a new skin to customize the appearance of {productname} {productmajorversion}.
 
 == Prerequisites
 
-This guide assumes:
+Before beginning, ensure that:
 
-* Familiarity with the command line and running simple commands.
-* https://nodejs.org/en/[Node.js], https://yarnpkg.com/en/[Yarn] and https://gulpjs.com[Gulp] are already installed.
-* Basic understanding of http://lesscss.org[Less], the CSS preprocessor we use to build the skins. More specifically, http://lesscss.org/features/#variables-feature[read the section about variables].
+* Familiarity with the command line and basic commands is established.
+* link:https://nodejs.org/en/[Node.js], link:https://yarnpkg.com/en/[Yarn], and link:https://gulpjs.com[Gulp] are already installed.
+* A basic understanding of link:http://lesscss.org[Less], the CSS preprocessor used to build skins, is acquired. Specifically, review the link:http://lesscss.org/features/#variables-feature[section on variables].
 
 == Preparation
 
-The CSS that goes with a theme is called a skin. The default skin for {productname} {productmajorversion} is named Oxide and is written in http://lesscss.org[Less], a popular CSS preprocessor. With Oxide we introduced a concept we call the *Style API*. This API consists of around 300 variables which you use to modify the appearance of {productname}. You never touch the underlying CSS. The benefit of this approach is that improvements we make to the CSS and HTML won't break your custom skin. This also means that if things don't work as expected, we can provide support and bug fixes, something that was virtually impossible before.
+The CSS associated with a theme is called a skin. The default skin for {productname} {productmajorversion} is **Oxide**, written in link:http://lesscss.org[Less]. Oxide introduces the *Style API*, which consists of approximately 300 variables that allow customization of {productname} without modifying the underlying CSS. This approach ensures compatibility with future updates and provides continued support for bug fixes.
 
-IMPORTANT: We do not recommend modifying or overriding CSS rules directly.
+[IMPORTANT]
+Do not modify or overriding CSS rules directly.
```

Comment:
Well, they can if they need to, it's not a strictly forbidden.

Ideally we want them to use as much variable as possible, but any more detailed modifications will require overwrites to CSS

Reply:
```suggestion
[IMPORTANT]
{companyname} recommends not modifying or overriding CSS rules directly.
```